iron-list.html

```diff
@@ -976,7 +976,7 @@
         var ts = window.performance.now();
         // Concat arrays in place.
         [].push.apply(this._physicalItems, this._createPool(delta));
-        [].push.apply(this._physicalSizes, new Array(delta));
+        [].push.apply(this._physicalSizes, new Array(delta).fill(0));
```

Comment:
So IE doesn't support `.fill()`, hence the test failures. LGTM otherwise.

Reply:
lol, yeah I was mislead by not seeing any red color in the MDN support table :p
Fixed in https://github.com/PolymerElements/iron-list/pull/491/commits/3772d0937fde34d2169335188bb5306b0fb82c76